recipes/remotezip/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "remotezip" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/remotezip-{{ version }}.tar.gz
+  sha256: ce65b7910c5c25d8950ed402023592967f5791ac14987141c050016ffad18dec
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Since this is pure Python, it can be noarch:
```suggestion
build:
  script: {{ PYTHON }} -m pip install . -vv
  number: 0
  noarch: python
```